lib/plane.rb

```diff
@@ -1,2 +1,30 @@
 class Plane
-end
+
+	DEFAULT_STATUS = "flying"
+
+	attr_accessor :status
+
+	def initialize
+		self.status = DEFAULT_STATUS
+	end
+
+	def land
+		self.status = "landed"
+	end
+
+	def landed? 
+ 		if self.status == "landed"
+ 			true
+ 		end
+ 	end
+
+ 	def take_off
+ 		self.status = "flying"
+ 	end
+
+	def took_off?
+		if self.status == "flying"
```

Comment:
Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression.<br>Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.